Shift3/standards-and-practices
# fix(#368): Fix moves View On Github button below the sidebar nav

## Changes

1. Adds custom styles
2. Updates dependencies

## Purpose

The View On Github button was covering the content. [Issue 358](https://github.com/Shift3/standards-and-practices/issues/358)

## Approach

Adds custom CSS to override theme button styles

## Pre-Testing TODOs

- Clone this branch
- Make sure you have Ruby and Jekyll installed
- Run `bundle install`

## Testing Steps

- Run `bundle exec jekyll serve` to host the site locally and view the changes

## Learning

Troubleshoot how the styles are being rendered on the gh-pages site. Find documentation on how to add custom styles to override the leap-day theme. Find out how to test the site locally to verify your changes are working before you push up the changes. Troubleshoot any errors encountered when trying to serve the site locally.

- [How to test your site locally with Jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)
- [Review the leap-day theme](https://pages-themes.github.io/leap-day/)
